validation/stratified_couette_flow/stratified_couette_flow.jl

```diff
@@ -11,7 +11,7 @@ using Oceananigans.Advection: cell_advection_timescale
 
 """ Friction velocity. See equation (16) of Vreugdenhil & Taylor (2018). """
 function uτ(model, Uavg, U_wall, n)
-    Nz, Hz, Δz = model.grid.Nz, model.grid.Hz, model.grid.Δzᵃᵃᶜ
+    Nz, Hz, Δz = model.grid.Nz, model.grid.Hz, model.grid.z.Δzᶜ
```

Comment:
```suggestion
    Nz, Hz, Δz = model.grid.Nz, model.grid.Hz, model.grid.z.Δzᵃᵃᶜ
```

I'm not sure we want to change the notation. Dropping the "a" has produced confusion in the past.